src/spi.rs

```diff
@@ -1168,16 +1168,12 @@ macro_rules! spi {
 
                     /// Internal implementation for blocking::spi::Transfer and
                     /// blocking::spi::Write
```

Comment:
This comment is no longer true: Transfer and Write now have separate internal functions.
Which makes me wonder: Is it still useful to have those internal functions, or could they be inlined into Transfer::transfer and Write::write?
(Keeping them separate could be useful if they can be reused to implement embedded-hal 1.0.0 traits.)

Reply:
It looks like they could be useful to implement `embedded-hal 1.0.0` traits `SpiBus`, `SpiBusRead` and `SpiBusWrite` (https://docs.rs/embedded-hal/1.0.0-alpha.8/embedded_hal/spi/blocking/index.html). In the meantime, I agree inlining into the traits would be cleaner